docs/05-PipelineLaunchOptions.md

```diff
@@ -105,6 +105,20 @@ Example for two entries/scans to be uploaded (one human subject and one phantom)
 
 ## 5.2 - Pipeline Launch for BIDS datasets
 
+### 5.2.1 - Ideal BIDS structure for LORIS
+
+The BIDS dataset to import has to:
+
+- pass the BIDS validator of [pybids](https://github.com/bids-standard/pybids)
+- contain a `participants.tsv` file at its root. Ideally, this file will contain the following columns:
+  - `participant_id`: the PSCID of the candidate  
+  - `date_of_birth`: the date of birth (in YYYY-MM-DD) to use when creating the candidate in LORIS 
+  - `sex`: the sex to use when creating the candidate in LORIS
+  - `site`: the name of the LORIS site to associate the candidate with when creating the candidate in LORIS
```

Comment:
Given Alfredo's feedback it might be good to add something like: This field is optional. If provided, the entry should match the name of a site already populate in the database. If missing or invalid, NULL will be used instead.

Reply:
thanks! I modified the text. Question: is it using the alias of the site or the actual name of the site for the site field? I think it is the site MRI alias field but am not sure.